client/src/components/CookieAlert.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import CloseX from "../components/Icon/close-button";
+import { theme } from "../constants/theme";
+
+const { colors } = theme;
+
+const CookieContainer = styled.div`
+  alignitems: center;
+  background: ${colors.darkerGray};
+  color: ${colors.selago};
+  display: flex;
+  font-family: Poppins;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  justify-content: center;
+  padding: 1.5rem 2rem;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  text-align: left;
+`;
+
+const CookieLink = styled.a`
+  color: ${colors.selago};
+  text-decoration: underline;
+`;
+
+const CloseButton = styled.button`
+  align-self: start;
+`;
+
+export default () => {
+  const [active, setActive] = useState();
```

Comment:
I think here we can set `active` initially as `true`, and then on line 38, we can replace the one-liner if-statement with `setActive(false)`,  since the the primary function of `hideMessage` is to handle the close button.